app/package.json

```diff
@@ -87,6 +87,7 @@
     "build:relay": "relay-compiler",
     "test": "jest --config ./jest.config.js",
     "dev": "pnpm run dev:server & pnpm run build:static && pnpm run build:relay && vite",
+    "dev:auth": "export DANGEROUSLY_SET_PHOENIX_ENABLE_AUTH=true && export DANGEROUSLY_SET_PHOENIX_SECRET=secret && pnpm run dev:server & pnpm run build:static && pnpm run build:relay && vite",
```

Comment:
lol you don't have this on all the time?

Reply:
no, do you? haha